plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreDataType.java

```diff
@@ -532,6 +533,25 @@ public DBSEntityType getEntityType() {
         return DBSEntityType.TYPE;
     }
 
+    @NotNull
+    @Override
+    public List<? extends DBSContextBoundAttribute> bindAttributesToContext(
+        @NotNull DBRProgressMonitor monitor,
+        @NotNull DBSEntity dataContainer,
+        @NotNull DBSEntityAttribute memberContext
+    ) throws DBException {
+        List<PostgreDataTypeAttribute> attrs = this.getAttributes(monitor);
+        if (attrs == null) {
+            return null;
```

Comment:
But you have @NotNull annotation on this method. Chose one something (:

Reply:
yeah, thanks